docs/UserGuide.md

```diff
@@ -337,7 +349,7 @@ from NUSMods database directly and displayed in the result display box.
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Note:**<br>
-If your internet is down, you can only search for modules that are of the Computer Science curriculum.
+If your internet is down, you can only search for modules that are in the NUS Computer Science curriculum.
```

Comment:
minor minor nit: can change to "not connected to internet" as you did earlier HAHAH